module/module_calls.go

```diff
@@ -31,20 +32,26 @@ type DeclaredModuleCall struct {
 	SourceAddr ModuleSourceAddr
 	Version    version.Constraints
 	InputNames []string
+	Range      *hcl.Range
```

Comment:
```suggestion
	RangePtr      *hcl.Range
```
This is just to follow a naming convention set by HCL itself, where in most cases `hcl.Range`s are not pointers and when they are, it's made clear(er) by that `Ptr` suffix.